### PR TITLE
[Added] Events and Groups endpoints

### PR DESCRIPTION
**Pull request by lazycipher**
I've added the following events endpoints:
Add Events:  POST `events/add`
Delete Events: DELETE `events/:eventId/delete`
List Participants: GET `events/:evendId/participants`
Register for Event: PUT `events/:eventId/register`

and the following group endpoints:
Add Group: POST `/groups/add`

### CODE REVIEW

File: src/routes/events.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        res.status(500).send('Server Error');
+    }
+});
+
+// @route    GET api/events/:eventId/participants
+// @desc     fetch all participants of an event
+// @access   Public
+router.get('/:evendId/participants', async(req, res)=> {
+    try{
+        const eventParticipants = await Events.findOne({uid: req.params.eventId});
```
Comment (honeycoder96) [inline src/routes/events.js]: findOne on event should be on eid i guess

File: src/routes/groups.js [hunk 2]
```diff
@@ -12,4 +18,25 @@ router.get('/', async(req, res)=> {
...
+// @access   Private: Only admins can add events
+router.post('/add', auth, async(req, res)=> {
+    const group = sanitize(req.body);
+    try{
+        const user = await User.findOne({ uid: req.user.uid }).select('-password');
+        if (user.role != 'admin') res.status(401).json({ msg: 'Not authorized' });
+        else {
+            let groupCreation = await Group.create(group);
+            if(groupCreation){
+                return res.status(201).json({ msg: 'Event Added Successfully' });
```
Comment (honeycoder96) [inline src/routes/groups.js]: Should be Group added instead of event

**Review by honeycoder96 — CHANGES_REQUESTED** [review]
Just few minor changes have a look at them.

File: src/routes/events.js (hunk 1 above)
Comment (lazycipher) [reply src/routes/events.js]: oh! my bad! I've changed that!

**Comment by lazycipher** [comment]
@honeycoder96 Can you check that again if everything's okay now, please?

**Review by honeycoder96 — APPROVED** [review]

**Pull request merged**